src/Api/Auth/Models/Request/Accounts/UpdateTdeOffboardingPasswordRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Api.Models.Request.Organizations;
+
+namespace Bit.Api.Auth.Models.Request.Accounts;
+
+public class UpdateTdeOffboardingPasswordRequestModel : OrganizationUserResetPasswordRequestModel
```

Comment:
I don't really like us extending an Admin Console model for this, can we just create our own?

Reply:
Agreed, I pulled in the properities from the `OrganizationUserResetPasswordRequestModel`, so now there is no coupling to it anymore.